publication-model/src/main/java/no/unit/nva/model/associatedartifacts/AssociatedArtifactList.java

```diff
@@ -13,14 +15,25 @@
 
 public class AssociatedArtifactList implements List<AssociatedArtifact> {
 
-    public static final String NULL_OBJECT_MUST_BE_SINGLETON_IN_LIST = "AssociatedArtifactLists containing "
-            + "NullAssociatedArtifact must contain only this element as a singleton";
     private final List<AssociatedArtifact> associatedArtifacts;
     
     @JsonCreator
     public AssociatedArtifactList(List<AssociatedArtifact> artifacts) {
-        throwExceptionIfNullObjectIsNotOnlyElementInList(artifacts);
-        this.associatedArtifacts = nonNull(artifacts) ? artifacts : Collections.emptyList();
+        if (isNull(artifacts)) {
+            this.associatedArtifacts = Collections.emptyList();
```

Comment:
Remove this and add the check to the getter

Reply:
Or do we want the empty list in ddb?

Reply:
It was empty list before also. We do not have getter here, we are in constructor.  Or what do you think?
```
    public AssociatedArtifactList(List<AssociatedArtifact> artifacts) {
        throwExceptionIfNullObjectIsNotOnlyElementInList(artifacts);
        this.associatedArtifacts = nonNull(artifacts) ? artifacts : Collections.emptyList();
   }
```

Reply:
For me, I don't think. persisting an empty or null value makes sense if we can deserialize it to an empty list. by adding it to a getter